Makefile

```diff
@@ -8,7 +8,7 @@ IS_64_BIT ?= $(shell python -c 'from sys import maxsize; print(maxsize > 2**32)'
 SAMPLE_BALLOT_COUNT ?= 5
 SAMPLE_BALLOT_SPOIL_RATE ?= 50
 
-all: environment install validate lint coverage
+all: environment install validate auto-lint coverage
```

Comment:
@keithrfung / @AddressXception - any objection to running the auto-lint here?

Reply:
No. That's good for my perspective. Makes sense.

Reply:
im in favor too